README.md

```diff
@@ -28,7 +28,7 @@ https://github.com/cockroachdb/cockroach#design).
 ## Status
 
 CockroachDB is currently in beta. See our
-[Roadmap](https://github.com/cockroachdb/cockroach/wiki) and
+[Roadmap](https://github.com/cockroachdb/cockroach/issues/12854) and
```

Comment:
Might want to call this the 1.0 Roadmap instead of just the Roadmap. That may help us remember to change the link once we get past 1.0.